packages/@uppy/image-editor/src/ImageEditor.tsx

```diff
@@ -46,24 +108,26 @@ export default class ImageEditor extends UIPlugin {
       quality: 0.8,
     }
 
+    const cropperOptions = {
+      ...defaultCropperOptions,
+      ...opts?.cropperOptions,
+    } as Cropper.Options
+
```

Comment:
Could we minimize the diff and not define a separate variable for that?
```suggestion
```

Reply:
Unfortunately TS is not smart enough to know that these two combined results in those options always being defined